<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>643</width>
    <height>539</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Add a plot configuration</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Source</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QComboBox" name="_source"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="label_2">
       <property name="text">
        <string>Name</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="_name">
       <property name="text">
        <string>New plot</string>
       </property>
      </widget>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="label_3">
       <property name="text">
        <string>Type</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QComboBox" name="_type">
       <item>
        <property name="text">
         <string>Stratigraphic</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Log measures continuous</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>Log measures instantaneous</string>
        </property>
       </item>
      </widget>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="label_4">
       <property name="text">
        <string>Feature reference column</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QComboBox" name="_feature_ref_column"/>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QStackedWidget" name="_type_widgets">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="page">
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <layout class="QFormLayout" name="formLayout_2">
         <item row="0" column="0">
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Depth from column</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QComboBox" name="_depth_from_column"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="_depth_from_column_2">
           <property name="text">
            <string>Depth to column</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="_depth_to_column"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="_depth_from_column_3">
           <property name="text">
            <string>Formation code column</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="_formation_code_column"/>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="_depth_from_column_4">
           <property name="text">
            <string>Rock code column</string>
           </property>
          </widget>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="_rock_code_column"/>
         </item>
         <item row="4" column="0">
          <widget class="QLabel" name="_depth_from_column_5">
           <property name="text">
            <string>Formation description column</string>
           </property>
          </widget>
         </item>
         <item row="4" column="1">
          <widget class="QComboBox" name="_formation_description_column"/>
         </item>
         <item row="5" column="0">
          <widget class="QLabel" name="_depth_from_column_6">
           <property name="text">
            <string>Rock description column</string>
           </property>
          </widget>
         </item>
         <item row="5" column="1">
          <widget class="QComboBox" name="_rock_description_column"/>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_2">
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QFormLayout" name="formLayout_3">
         <item row="0" column="0">
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Unity of measure</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="_uom_continuous"/>
         </item>
         <item row="1" column="0">
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Start measure column</string>
           </property>
          </widget>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Interval column</string>
           </property>
          </widget>
         </item>
         <item row="3" column="0">
          <widget class="QLabel" name="label_8">
           <property name="text">
            <string>Value column</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="_start_measure_column"/>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="_interval_column"/>
         </item>
         <item row="3" column="1">
          <widget class="QComboBox" name="_value_continuous_column"/>
         </item>
         <item row="4" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QCheckBox" name="_continuous_min_chk">
             <property name="text">
              <string>Min value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="_continuous_min">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="_continuous_max_chk">
             <property name="text">
              <string>Max value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="_continuous_max">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="page_3">
      <layout class="QVBoxLayout" name="verticalLayout_4">
       <item>
        <layout class="QFormLayout" name="formLayout_4">
         <item row="1" column="0">
          <widget class="QLabel" name="label_9">
           <property name="text">
            <string>Event column</string>
           </property>
          </widget>
         </item>
         <item row="1" column="1">
          <widget class="QComboBox" name="_event_column"/>
         </item>
         <item row="2" column="0">
          <widget class="QLabel" name="label_10">
           <property name="text">
            <string>Value column</string>
           </property>
          </widget>
         </item>
         <item row="2" column="1">
          <widget class="QComboBox" name="_value_instantaneous_column"/>
         </item>
         <item row="0" column="0">
          <widget class="QLabel" name="label_12">
           <property name="text">
            <string>Unity of measure</string>
           </property>
          </widget>
         </item>
         <item row="0" column="1">
          <widget class="QLineEdit" name="_uom_instantaneous"/>
         </item>
         <item row="3" column="0" colspan="2">
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QCheckBox" name="_instantaneous_min_chk">
             <property name="text">
              <string>Min value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="_instantaneous_min">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QCheckBox" name="_instantaneous_max_chk">
             <property name="text">
              <string>Max value</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="_instantaneous_max">
             <property name="enabled">
              <bool>false</bool>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_continuous_min_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>_continuous_min</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>51</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>208</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_continuous_max_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>_continuous_max</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>481</y>
    </hint>
    <hint type="destinationlabel">
     <x>524</x>
     <y>482</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_instantaneous_min_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>_instantaneous_min</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>61</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>213</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>_instantaneous_max_chk</sender>
   <signal>toggled(bool)</signal>
   <receiver>_instantaneous_max</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>366</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>519</x>
     <y>266</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
